src/Buy.vue

```diff
@@ -112,7 +112,7 @@ export default {
             eventLabel: JSON.stringify(err),
           });
 
-          if (err.message && err.message.indexOf('User denied transaction signature.') !== -1)  {
```

Comment:
Is the stack, a stack of messages? Should the indexof search for partial matches then, or are we sure that's the full string?

Reply:
It used to live in `err.message`, now they switched it to a generic JSONRPC "internal server error" message and put extra info in a string in `err.stack`.